### PR TITLE
PIM-3849 + PIM-3880: Fix bad completeness scope on mass edit actions

### PR DESCRIPTION
**Pull request by grena**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | Y
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | Y
| Fixed tickets        | PIM-3849, PIM-3880
| DB schema updated?   | N
| Migration script?    | -
| Doc PR               | -

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Datagrid/Product/ContextConfigurator.php [hunk 1]
```diff
@@ -317,13 +317,25 @@ protected function getCurrentLocaleCode()
...
-            return $filterValues['scope']['value'];
-        } else {
-            $channel = $this->userContext->getUser()->getCatalogScope();
+            $currentScopeCode = $filterValues['scope']['value'];
+        }
 
-            return $channel->getCode();
+        if (null === $currentScopeCode) {
+            $requestFilters = $this->request->get('filters');
+            if (isset($requestFilters['scope']['value']) && $requestFilters['scope']['value'] !== null) {
```
Comment (solivier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Product/ContextConfigurator.php]: isset($requestFilters['scope']['value']) && null !== $requestFilters['scope']['value']

File: src/Pim/Bundle/DataGridBundle/Datagrid/Product/ContextConfigurator.php [hunk 2]
```diff
@@ -317,13 +317,25 @@ protected function getCurrentLocaleCode()
...
+        $currentScopeCode = null;
 
-            return $channel->getCode();
+        if (isset($filterValues['scope']['value']) && null !== $filterValues['scope']['value']) {
+            $currentScopeCode = $filterValues['scope']['value'];
+        }
+
+        if (null === $currentScopeCode) {
+            $requestFilters = $this->request->get('filters');
+            if (isset($requestFilters['scope']['value']) && null !== $requestFilters['scope']['value']) {
```
Comment (jjanvier) [inline src/Pim/Bundle/DataGridBundle/Datagrid/Product/ContextConfigurator.php]: no need to test it's `null !==`, you already did it with `isset`
